src/utils/utils.py

```diff
@@ -551,10 +576,31 @@ def is_workload_empty(path):
     if os.path.isfile(pmc_perf_path):
         temp_df = pd.read_csv(pmc_perf_path)
         if temp_df.dropna().empty:
-            error(
-                "[profiling] Error. Found empty cells in %s.\nProfiling data could be corrupt."
+            console_error(
+                "profiling"
+                "Found empty cells in %s.\nProfiling data could be corrupt."
                 % pmc_perf_path
             )
 
     else:
-        error("[profiling] Error. Cannot find pmc_perf.csv in %s" % path)
+        console_error("profiling", "Cannot find pmc_perf.csv in %s" % path)
+
+
+def print_status(msg):
+    msg_length = len(msg)
+
+    console_log("")
+    console_log("~" * (msg_length + 1))
+    console_log(msg)
+    console_log("~" * (msg_length + 1))
+    console_log("")
+
+
+def set_locale_encoding():
+    try:
+        locale.setlocale(locale.LC_ALL, "en_US.UTF-8")
+    except locale.Error as error:
+        print("Please ensure that the 'en_US.UTF-8' locale is available on your system.")
+        print("")
+        print("ERROR: ", error)
+        sys.exit(1)
```

Comment:
Is there a reason we're not using `console_error()` here?  If we want to standardize it we can use:
```suggestion
        console_log("Please ensure that the 'en_US.UTF-8' locale is available on your system.")
        console_log("")
        console_error(error)
```

Reply:
Yes, because this check happens right up front before the logging handlers and formatters are configured. Consequently, you will get the default formatter which is a bit confusing and looks odd.

To be fair, I'm not sure we need this check right up front but I was trying to keep it similar to what you had while moving out of top-level omniperf to keep that file clean.  If we wanted to push this check a bit further down the call stack, then we could likely go back to using `console_error()`